Workbooks/Virtual Machines - Network Dependencies/Connections Overview/Connections Overview.workbook

```diff
@@ -144,9 +98,7 @@
       "content": {
         "version": "KqlParameterItem/1.0",
         "query": "",
-        "crossComponentResources": [
-          "{Workspace}"
-        ],
+        "crossComponentResources": [],
         "parameters": [
           {
             "id": "306614b4-14f2-4777-a044-8c1121f04d6d",
```

Comment:
If you are not planning on keeping "GridRefresh" please remove it

Reply:
Done